qdmr_parsing/model/run_model.py

```diff
@@ -94,10 +105,14 @@ def validate_args(args):
     if args.model in ["seq2seq", "copynet", "dynamic"]:
         assert os.path.exists(args.model_dir)
 
-    # seq2seq dynamic only accepts input file at the moment
-    # TODO: add option for single example prediction
-    if args.model == "dynamic":
-        assert args.input_file
+
+def get_allowed_tokens(valid_tokens):
```

Comment:
This functionality is implemented here:
https://github.com/tomerwolgithub/Break/blob/master/qdmr_parsing/utils/preprocess_examples.py

Could you please update the code to use it instead?